frontend/lib/api/form/pollingstation/pollingStationUtils.ts

```diff
@@ -294,3 +295,104 @@ function sortFormSections(a: FormSection, b: FormSection): number {
   if (a.index > b.index) return 1;
   return 0;
 }
+
+export function getInitialValues(election: Required<Election>, defaultValues?: Partial<PollingStationValues>) {
```

Comment:
probably implied, but perhaps add a return type to result or function?
`getInitialValues(...): PollingStationValues`

Reply:
That's indeed already the implicit (inferred) type, but I've made it explicit in f8fc6ca3aaaba794243d95f043cbf1fd2d6c1d94.